sql/deparse.sql

```diff
@@ -0,0 +1,6 @@
+-- This file and its contents are licensed under the Apache License 2.0.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-APACHE for a copy of the license.
+
+CREATE OR REPLACE FUNCTION _timescaledb_internal.deparse_table(tbl REGCLASS) RETURNS TEXT
```

Comment:
I suggest we name this `ts_get_tabledef` so that naming is analogous to `pg_get_indexdef`, `pg_get_constraintdef`, `pg_get_triggerdef`, etc.

Reply:
done

Reply:
did you push your changes. I don't see the name change.

Reply:
I just pushed my changes